internal/config/config.go

```
@@ -57,6 +57,7 @@ type Healthcheck struct {
 	Name            string        `yaml:"name" default:"healthcheck"`
 	Endpoint        string        `yaml:"endpoint" default:"/healthz"`
 	RefreshDuration time.Duration `yaml:"refreshDuration" default:"5s"`
+	MaskURLs        bool          `yaml:"maskURLs" default:"false"`
```

golangci-lint (2.12.2)

[error] yaml(camel): got 'maskURLs' want 'maskUrLs'

(tagliatelle)